OverlayPlugin.Core/MemoryProcessors/InCombat/InCombatMemoryManager.cs

```diff
@@ -18,19 +18,14 @@ class InCombatMemoryManager : IInCombatMemory
         public InCombatMemoryManager(TinyIoCContainer container)
         {
             this.container = container;
-            container.Register<IInCombatMemory60, InCombatMemory60>();
             container.Register<IInCombatMemory61, InCombatMemory61>();
             repository = container.Resolve<FFXIVRepository>();
         }
 
         private void FindMemory()
         {
             List<IInCombatMemory> candidates = new List<IInCombatMemory>();
-            // For CN, try the lang-specific candidate first, then fall back to intl
-            if (repository.GetMachinaRegion() == GameRegion.Korean)
-            {
-                candidates.Add(container.Resolve<IInCombatMemory60>());
-            }
+            // For CN/KR, try the lang-specific candidate first, then fall back to intl

```

Comment:
Remove this comment since it's no longer relevant, and delete `InCombatMemory60`.